test/javascripts/acceptance/encrypt-test.js

```diff
@@ -659,6 +659,302 @@ acceptance("Encrypt - active", function (needs) {
     );
   });
 
+  test("draft for new topics is encrypted", async function (assert) {
+    pretender.post("/drafts.json", (request) => {
+      const data = JSON.parse(parsePostData(request.requestBody).data);
+      if (data.title) {
+        assert.notStrictEqual(data.title, `Some public message ${PLAINTEXT}`);
+      }
+      if (data.reply) {
+        assert.notStrictEqual(
+          data.reply,
+          `Hello, world! ${PLAINTEXT}`.repeat(42)
+        );
```

Comment:
Just a suggestion:

Perhaps define variables for `title` and `text` to be used here and when filling in as well/

___

Separately, is there a need for the `if` statements?

Reply:
Looking at this again I realize that it might be better to check if `PLAINTEXT` is not included in `data.title` or `data.reply`

The if is necessary because a draft might be saved after the title was filled in, but reply has not been yet.